addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -248,7 +248,7 @@
                     <t t-if="availability_check" t-foreach="tickets" t-as="ticket">
                         <t t-foreach="range(1, ticket['quantity'] + 1)" t-as="att_counter" name="attendee_loop">
                             <t t-set="counter" t-value="counter + 1"/>
-                            <div class="modal-body">
+                            <div t-attf-class="#{'modal-body' if event.specific_question_ids else 'd-none'}">
```

Comment:
You could move the `<input class="d-none" ...` that is a few lines underneath, just above this div.
Then you can use `t-if="event.specific_question_ids"` on this div and remove the one line 253 :slightly_smiling_face: